wagtail/admin/tests/pages/test_create_page.py

```diff
@@ -1800,3 +1802,40 @@ def test_post_with_comment_notifications_switched_off(self):
 
         self.assertEqual(subscription.user, self.user)
         self.assertFalse(subscription.comment_notifications)
+
+
+class TestPagePickleSupport(WagtailTestUtils, TestCase):
```

Comment:
This test would be better placed in `wagtail.tests.test_streamfield` since it's not testing anything to do with the admin module (and the `self.login()` is redundant).

Reply:
Good point! I was looking for a proper place. I'll move it there.